ui-ngx/src/app/modules/home/components/widget/lib/gateway/connectors-configuration/type-value-panel/type-value-panel.component.html

```diff
@@ -54,13 +54,24 @@
                 <div class="tb-form-row column-xs" fxLayoutAlign="space-between center">
                   <div class="fixed-title-width tb-required" translate>gateway.value</div>
                   <mat-form-field fxFlex appearance="outline" subscriptSizing="dynamic" class="tb-inline-field flex tb-suffix-absolute">
-                    <input matInput required formControlName="value"
-                           placeholder="{{ 'gateway.set' | translate }}"/>
+                    <ng-container [ngSwitch]="keyControl.get('type').value">
+                      <input *ngSwitchCase="MappingValueType.STRING" matInput required formControlName="string"
+                             placeholder="{{ 'gateway.set' | translate }}" />
+                      <input *ngSwitchCase="MappingValueType.INTEGER" matInput required formControlName="integer" type="number"
+                             placeholder="{{ 'gateway.set' | translate }}" />
+                      <input *ngSwitchCase="MappingValueType.DOUBLE" matInput required formControlName="double" type="number"
+                             placeholder="{{ 'gateway.set' | translate }}" />
+                      <mat-select *ngSwitchCase="MappingValueType.BOOLEAN" formControlName="boolean">
+                        <mat-option [value]="true">true</mat-option>
+                        <mat-option [value]="false">false</mat-option>
+                      </mat-select>
```

Comment:
It might be better to replace it with `mat-slider`?

Reply:
Illia decided that option select is better